sympy/logic/boolalg.py

```diff
@@ -1584,7 +1593,7 @@ def conjuncts(expr):
 
 
 def disjuncts(expr):
-    """Return a list of the disjuncts in the sentence s.
+    """Return a list of the disjuncts in ``expr``.
```

Comment:
I think the style guide recommends using `*expr*` for function parameters.

Reply:
OK!

Reply:
I do not agree with this. 1) it looks bad, using double backticks makes it look like code and it is much more clear what one refer to, 2) it quite often says "if simplify=True" in the text and formatting that like `if *simplify*=...` doesn't make sense.

Reply:
I'm not sure where the `*param*` recommendation came from